client/configuration.py

```diff
@@ -131,17 +132,24 @@ def __init__(
         self._typeshed = None  # type: Optional[str]
 
         # Handle search path from multiple sources
-        self._search_path = []
-        pythonpath = os.getenv("PYTHONPATH")
-        if preserve_pythonpath and pythonpath:
-            for path in pythonpath.split(":"):
-                if os.path.isdir(path):
-                    self._search_path.append(SearchPathElement(path))
-                else:
-                    LOG.warning(
-                        "`{}` is not a valid directory, dropping it "
-                        "from PYTHONPATH".format(path)
-                    )
+        self.search_path = []
```

Comment:
Something went wrong with the rebase I think, you accidentally renamed `self._search_path` to `self.search_path` in a couple of places.